rosa_getting_started_sts/rosa-sts-deleting-access-cluster.adoc

```diff
@@ -5,8 +5,12 @@ include::modules/attributes-openshift-dedicated.adoc[]
 
 toc::[]
 
-Delete access to a {product-title} (ROSA) cluster using the `rosa` command-line.
+Access to a {product-title} (ROSA) cluster is controlled through the identity provider (IDP) accounts of users. To remove full access of a user to a cluster, you must delete the Identity Provider (IDP) account that was configured for the user.
+
+You can revoke the administrator access of users so that they can access the cluster without administrator privileges. To remove the administrator access for a user, you must revoke the `dedicated-admin` or `cluster-admin` privileges. You can revoke the administrator privileges using the `rosa` command-line utility or using the OpenShift Cluster Manager (OCM) console.
```

Comment:
this is confusing here since we are talking about revoking access to a cluster completely . We should have another section that introduces the topic of removing elevated access, then it delves into the different ways to do it.

Reply:
@0kashi: The way the topic is currently structured, "Revoking dedicated-admin access" and "Revoking cluster-admin access" are separate modules. Per the peer feedback, the recommendation was to keep them that way based on the modular documentation guidelines. While this method would include a few duplicated content in the topic, but it allows the procedures to be used separately. I'll check with @nermina-redhat when we're back from the year-end holidays and make the necessary changes. I've incorporated the rest of the feedback.

Reply:
We can keep them as separate modules.  I just think that including information about removing elevated privileges under the title of "Deleting access to a ROSA cluster" is confusing.  Let's have another title for "Revoking elevated access" (or something similar) and put the two modules underneath that.

Reply:
Made the changes to the section titles to reflect this feedback.